crates/sui-source-validation-service/README.md

```diff
@@ -0,0 +1,76 @@
+# Sui Source Validation Service
+
+This document describes the Sui Source Validation Service. It is engineering documentation primarily for MystenLabs engineers who may want to build, extend, configure, or understand the service.
+
+The Source Validation Service is a server that returns Move source code associated with on-chain Move bytecode. It fetches and builds Move source code for for a repository, and then verifies that the built artifact matches the on-chain bytecode. 
+
+The default configuration limits scope to Sui framework packages in `crates/sui-framework/packages`:
+
+- `move-stdlib` — [address `0x1`](https://suiexplorer.com/object/0x1)
+- `sui-framework` — [address `0x2`](https://suiexplorer.com/object/0x2)
+- `sui-system` — [address `0x3`](https://suiexplorer.com/object/0x2)
+- `deepbook` — [address `0xdee9`](https://suiexplorer.com/object/0xdee9)
+
+See examples below for requesting source from the server.
+
+## Build and Run
+
+```
+cargo build --release --bin sui-source-validation-service
+```
+
+```
+cargo run --bin sui-source-validation-service crates/sui-source-validation-service/config.toml 
+```
+
+See [`config.toml` in this directory](./sui-source-validation-service/config.toml).
```

Comment:
Will this link work? It seems like the README is in the same directory so it should just be:

```suggestion
See [`config.toml` in this directory](./config.toml).
```

Reply:
It should--for some reason trying this within GHs markdown editor didn't. I think that's a bug, it works fine in the branch view of the README, so doing this.